packages/x-charts-pro/src/FunnelChart/FunnelPlot.tsx

```diff
@@ -20,6 +20,11 @@ export interface FunnelPlotProps extends FunnelPlotSlotExtension {
    * @default 0
    */
   gap?: number;
+  /**
+   * The radius, in pixels, of the corners of the funnel sections.
+   * @default 0
+   */
+  borderRadius?: number;
```

Comment:
@mui/xcharts should this be in the series instead? 🤔 
It is an easy change.

The gap would look weird on per-series basis. But the borderRadius should work just fine.